graphene_sqlalchemy/tests/test_converter.py

```diff
@@ -336,6 +337,22 @@ class TestEnum(enum.IntEnum):
     assert graphene_type._meta.enum.__members__["two"].value == 2
 
 
+@pytest.mark.skipif(
+    not SQL_VERSION_HIGHER_EQUAL_THAN_1_4,
+    reason="SQLAlchemy <1.4 does not support this",
+)
+def test_should_columproperty_convert_sqa_20():
```

Comment:
NIT: You could hand the select function and parametrize the test depending on the SQL version 🙂

Reply:
Good Idea!